src/schema/datastore-schema.js

```diff
@@ -15,6 +15,7 @@ create table if not exists record (
   record text,
   parentId varchar(60),
   timestamp BIGINT NOT NULL,
+  recordTimestamp DATETIME NOT NULL,
```

Comment:
The initial schema should not be changed. The schema should only be changed with migrations to ensure that the migrations have a known previous database state to migrate from.